dags/data_package_generator.py

```diff
@@ -0,0 +1,82 @@
+from datetime import timedelta
+
+from airflow import DAG
+
+from utils import dag_default_args, get_config, setup_configure_dag_callable
+from airflow.providers.amazon.aws.operators.ecs import (
+    EcsRegisterTaskDefinitionOperator,
+    EcsRunTaskOperator,
+)
+from airflow.operators.python import PythonOperator
+from airflow.models.param import Param
+
+data_packages = ["organisation"]
+
+# read config from file and environment
+config = get_config()
+
+# set some variables needed for ECS tasks,
+ecs_cluster = f"{config['env']}-cluster"
+task_definition_name = f"{config['env']}-data-package-builder-task"
+
+for package in data_packages.items():
+    with DAG(
+            f"{package}-builder",
+            default_args=dag_default_args,
+            description=f"Data package builder task for the {package} data package",
+            schedule=None,
+            catchup=False,
+            params={
+                "cpu": Param(default=8192, type="integer"),
+                "memory": Param(default=32768, type="integer"),
+                "transformed-jobs": Param(default=8, type="integer"),
```

Comment:
These are specific to the collection task so while they can be fed in without causing problems they aren'r rally doing much here. They essentially tell make how many parallel jobs to run at once.

Reply:
Cheers - removed